src/core/lib/support/wrap_memcpy.c

```diff
@@ -40,7 +40,7 @@
  */
 
 #ifdef __linux__
-#ifdef __x86_64__
+#if defined(__x86_64__) && defined(__GNU_LIBRARY__)
```

Comment:
It's actually kind of a tough question, because you need to compare memmove with glibc's memcpy prior to libc 2.14, which is the whole motivation for that piece of code here. And in all cases, it's still not going to help without a good solution for `sched_getcpu`, so...